### PR TITLE
Homework: Fibonacci

### PR DESCRIPTION
**Pull request by runer53248**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @runer53248

**Comment by ziobron** [comment]
⏱ 3 XP extra for @runer53248 for on-time delivery

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,23 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(sequence < 2){
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:4]: Add space after while, if, else etc...

File: homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+        return sequence;
+    }
+    int result = 0;
+    int prePreFibonaci = 0;
+    int preFibonaci = 1;
+    for(int i = 2; i <= sequence; ++i){
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:10]: Add space between operations like '+' '-' '!=' '==' etc...

Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:10]: Add space after while, if, else etc...
And also after `;`

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**